### PR TITLE
Figma tokens

### PR DESCRIPTION
**Pull request by taylornowotny**
color tokens update - added gradient colors and adjusted success colors

### CODE REVIEW

**Comment by rpominov** [comment]
Github action that converts `tokens.json` to `tokens.ts` fails. I need to fix it before I can merge. Going to fix on Monday

**Comment by rpominov** [comment]
@taylornowotny Typography tokens have broken links. That's why I can't merge this PR. Could you fix the links and sync?

Screenshot 2023-02-20 at 11 00 05

**Comment by kof** [comment]
Is there a way to report this error without debugging? Its going to happen from time to time

**Comment by rpominov** [comment]
> Is there a way to report this error without debugging? Its going to happen from time to time

I've made script output look like this now when there're broken links

```
Error: Broken links found:
typography > Regular > fontSize
typography > Labels - Title Case > fontSize
typography > Labels - Sentence case > fontSize
typography > Titles > fontSize
typography > Small > fontSize
typography > Tiny > fontSize
typography > Unit > fontSize
typography > Mono > fontWeight
typography > Mono > fontSize
typography > Big Title > fontSize
typography > Space Section Unit Text > fontWeight
typography > Space Section Unit Text > fontSize
typography > Space Section Value Text > fontSize
typography > Brand > Section Title > fontSize
typography > Brand > Large Title > fontSize
typography > Brand > regular > fontSize
typography > Brand > Thumbnail Large > Default > fontSize
typography > Brand > Thumbnail Large > Hover > fontSize
typography > Brand > Thumbnail Small > Default > fontSize
typography > Brand > Thumbnail Small > Hover > fontSize
typography > Brand > button > regular > fontSize
typography > Brand > button > cta > fontSize
```

**Comment by taylornowotny** [comment]
I re-synced everything and pushed to github. Did that fix the issue?

File: packages/design-system/src/__generated__/figma-design-tokens.ts
Comment (rpominov) [inline packages/design-system/src/__generated__/figma-design-tokens.ts:27]: @taylornowotny 

Can we use "line height", "font size", "font weight", and "letter spacing" tokens in code directly instead of via composed "typography" tokens? Or are these low level properties of text are only for Figma and shouldn't be used in code.

If we want to share them between code and Figma we need to be careful with changes like this.

**Comment by rpominov** [comment]
> I re-synced everything and pushed to github. Did that fix the issue?

Yes, thanks!

**Review by rpominov — APPROVED** [review]

**Pull request merged**
